docs/poly-apis.md

```diff
@@ -0,0 +1,636 @@
+# Poly RPCs
+
+## Table of Contents
+
+* [RPC Methods](#rpc-methods)
+    * [Method `poly_getCreatorId`](#method-poly_getCreatorId)
+    * [Method `poly_getDefaultFromId`](#method-poly_getDefaultFromId)
+    * [Method `poly_getContractValidatorTypeHash`](#method-poly_getContractValidatorTypeHash)
+    * [Method `poly_getRollupTypeHash`](#method-poly_getRollupTypeHash)
+    * [Method `poly_getEthAccountLockHash`](#method-poly_getEthAccountLockHash)
+    * [Method `poly_version`](#method-poly_version)
+
+* [RPC Types](#rpc-types)
+    * [Type `Uint32`](#type-uint32)
+    * [Type `Uint64`](#type-uint64)
+    * [Type `H256`](#type-h256)
+    * [Type `JsonBytes`](#type-jsonbytes)
+    * [Type `PolyVersionInfo`](#type-polyversioninfo)
+    * [Type `Versions`](#type-versions)
+    * [Type `NodeInfo`](#type-NodeInfo)
+    * [Type `RollupCell`](#type-rollupcell)
+    * [Type `RollupConfig`](#type-rollupconfig)
+    * [Type `GwScripts`](#type-gwscripts)
+    * [Type `EoaScripts`](#type-eoascripts)
+    * [Type `Backends`](#type-backends)
+    * [Type `Accounts`](#type-accounts)
+    * [Type `Script`](#type-scripts)
+    * [Type `ScriptHashType`](#type-scripthashtype)
+    * [Type `ScriptInfo`](#type-scriptinfo)
+    * [Type `BackendInfo`](#type-backendinfo)
+    * [Type `AccountInfo`](#type-accountinfo)
+
+## RPC Methods
+
+#### Method `poly_getCreatorId`
```

Comment:
It is worth adding a description to every RPC method, and explaining what the returned value indicates.

Reply:
Updated